### DISCUSSION TITLE
pivotwider and namesep

### DISCUSSION THREAD

**Discussion by arnaudcol** [pbeshai/tidy#74] [Q&A]
I have some data I am pivoting wide which has multiple values. I was assuming the nameSep configuration would handle the multiple values, but it doesn't seem to work.  It looks like I can only get the last value.
Any guidance on concatenating multiple values?
![image](https://github.com/user-attachments/assets/0df3de91-b785-4331-8452-e283bf1b319a)

**Comment by pbeshai** [comment]
Ah I suppose this would be similar to the `values_fn` option in tidyr. This is not currently supported, you'll need to summarize the data beforehand unfortunately
